README.md

```diff
@@ -109,6 +110,23 @@ or
 node index.js examples/bubble_sort.js examples/selection_sort.js
 ```
 
+### TOML Config
+
+You can create a [TOML](https://toml.io/en/) `.explainer-config.toml` config file that contains all of your options for the tool and place it in your `home` directory, i.e. (or simply modify and copy the one that is in the `examples directory`):
+```
+# Any of these config options can or can not be provided
+# Feel free to place this file in your home directory, i.e: cp examples/.explainer-config.toml ~
```

Comment:
I think it'd better to have this in the markdown as a bash code block opposed to in the file itself. Also, what if the user is not using a Linux system? How would they copy it over if they are using windows?

Reply:
updated

Reply:
As previously discussed,  The command works for `PS` and `WSL`(Linux environment) but it will fail fail `cmd` terminal. Explicit instructions should be provided for users with `cmd`  terminal.

Reply:
Added an example of how to copy the `.toml` file from `examples` folder into the `home` directory using `Windows` **CMD**